docker/scripts/docker_builder.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import argparse
+import json
+
+
+def parse_arguments(argv):
+    parser = argparse.ArgumentParser(description="Radameon docker build script")
+    parser.add_argument('-d', '--docker-context', help='Path to directory containing docker files (i.e. build context)')
+    parser.add_argument('-f', '--docker-file', default='', help='Path to specific docker file to build, if left empty, builds all of them')
+    parser.add_argument('-r', '--relation-file', default='', help='Path to relations file')
+
+    parsed_args = parser.parse_args(argv)
+
+    return parsed_args.docker_context, parsed_args.docker_file, parsed_args.relation_file
+
+
+def tag_image(image_digest, image_name, tag):
+    tag_cmd = ["docker", "tag", image_digest, "radameon/{}:{}".format(image_name, tag)]
+    ret = subprocess.run(tag_cmd)
+    if (ret.returncode == 0):
+        print("Image tagged successfully")
+        return True
+    else:
+        print("Couldn't tag docker image")
+        print(ret.args)
+        return False
+
+
+def build_file(context, file, image_config):
+    print("Building single docker file")
+
+    if not os.path.isfile(file):
+        print("Supplied file path is not an actual file")
+        return False
+
+    if not os.path.isdir(context):
+        print("Supplied context path is not a dockerfiles directory")
+        return False
+
+    build_cmd = []
+    if "args" in image_config.keys():
+        build_cmd = ["docker", "build", "--build-arg"]
+
+        for k in image_config["args"]:
+            build_cmd += [k + "=" + image_config["args"][k]]
+
+        build_cmd += ["-f", file, context]
+    else:
```

Comment:
`if...else` seems excessive here, you can relocate
```
 build_cmd += ["-f", file, context]
```
lower so under `if` you'll have only additions of args.